docs/application/tizen-studio/native-tools/ui-builder/component-attributes.md

```diff
@@ -1000,12 +1000,12 @@ The following attributes are used for the spinner component.
 | `max`      | 0 ~                                      | `elm_spinner_min_max_set()`          |
 | `disable`  | `true` or `false`                        | `elm_object_disabled_set()`          |
 | `visible`  | `true` or `false`                        | `evas_object_show()`                 |
-| `style`    | See the [supported properties](#supported_properties) for each profile and version | `elm_object_style_set()`             |
-| `colors`   | `#000000 ~ #ffffff`</br>See the [supported properties](#supported_properties) for each profile and version. | `elm_object_color_class_color_set()` |
+| `style`    | See the [supported properties](#properties-supported-for-each-component) for each profile and version | `elm_object_style_set()`             |
+| `colors`   | `#000000 ~ #ffffff`</br>See the [supported properties](#properties-supported-for-each-component) for each profile and version. | `elm_object_color_class_color_set()` |
 
 ### Toolbar (![Toolbar](./media/component_attributes_toolbar_icon.png)) in Mobile Applications
 
-A [toolbar](../../native/guides/ui/efl/component-toolbar-m.md) component is a small scrollable list of items. It can also show a menu when an item is selected. Only one item can be selected at a time.
+A [toolbar](../../native/guides/ui/efl/mobile/component-toolbar.md) component is a small scrollable list of items. It can also show a menu when an item is selected. Only one item can be selected at a time.
```

Comment:
It looks `../../../native/guides/ui/efl/mobile/component-toolbar.md` is correct.

Reply:
done